services-benchmarks/src/main/java/io/scalecube/services/benchmarks/transport/BenchmarkServiceState.java

```diff
@@ -27,12 +30,24 @@ public BenchmarkServiceState(BenchmarkSettings settings, Object... services) {
 
   @Override
   public void beforeAll() {
-    seed = Microservices.builder().metrics(registry()).startAwait();
+    seed =
+        Microservices.builder()
+            .metrics(registry())
+            .discovery(
+                (serviceRegistry, serviceEndpoint) ->
```

Comment:
it looks so convoluted, do we have another way to specify necessary options?

Reply:
i agree maybe its not new functionalty but it offers new convoluted feature

Reply:
@segabriel 
> it looks so convoluted, do we have another way to specify necessary options?

i will address only constructive criticism